vscode/src/non-stop/FixupController.ts

```diff
@@ -681,12 +702,18 @@ export class FixupController
         })
 
         if (!editOk) {
-            telemetryService.log('CodyVSCodeExtension:fixup:revert:failed')
+            telemetryService.log('CodyVSCodeExtension:fixup:revert:failed', { hasV2Event: true })
+            telemetryRecorder.recordEvent('cody.fixup.revert', 'failed')
             return
         }
 
         const tokenCount = countCode(replacementText)
-        telemetryService.log('CodyVSCodeExtension:fixup:reverted', tokenCount)
+        telemetryService.log('CodyVSCodeExtension:fixup:reverted', tokenCount, { hasV2Event: true })
+        const { metadata, privateMetadata } = splitSafeMetadata(tokenCount)
+        telemetryRecorder.recordEvent('cody.fixup.reverted', 'clicked', {
+            metadata,
+            privateMetadata,
+        })
```

Comment:
tokenCount already only has numeric values:

```suggestion
        telemetryService.log('CodyVSCodeExtension:fixup:reverted', tokenCount, { hasV2Event: true })
        telemetryRecorder.recordEvent('cody.fixup.reverted', 'clicked', {
            metadata: tokenCount,
        })
```